test/solidity/Facets/CBridge.t.sol

```diff
@@ -195,6 +203,87 @@ contract CBridgeFacetTest is TestBaseFacet {
         vm.stopPrank();
     }
 
+    function test_triggerRefund_Success() public {
+        address callTo = address(CBRIDGE_ROUTER);
+        bytes memory callData = abi.encodeWithSignature("someFunction()");
+        address assetAddress = ADDRESS_USDT;
+        address to = USER_RECEIVER;
+        uint256 amount = 100 * 10 ** usdt.decimals();
+
+        deal(ADDRESS_USDT, address(cBridge), amount);
+
+        vm.mockCall(callTo, callData, abi.encode(true));
+
+        vm.expectEmit(true, true, true, true, address(cBridge));
+        emit CBridgeRefund(assetAddress, to, amount);
+
+        cBridge.triggerRefund(
```

Comment:
in general when you have a token transfer involved in the function it also makes sense to check the balance changes in the calling wallet to ensure all aspects of the function work as expected